hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/aclDrawer/aclDrawer.tsx

```diff
@@ -70,7 +71,6 @@ export class AclPanel extends React.Component<IAclDrawerProps> {
           closable={false}
           visible={visible}
           getContainer={false}
-          style={{position: 'absolute'}}
```

Comment:
Is this removed due to the eslint?

Reply:
Hi, I might have removed it as styles were not supported in AntD v5 in this manner.
Thanks for pointing this out. Added the style back in the latest commit.

Reply:
I see, we can remove it anyway if it's not used in the AntD v5. Or you can file a follow up ticket. Just making sure that the all the changes are related to the dependency migrations.